release_docs/RELEASE_PROCESS.md

```diff
@@ -114,7 +114,7 @@ For more information on the HDF5 versioning and backward and forward compatibili
 7. Choose the release branch
 8. Change ‘Release version tag’ name to 'hdf5_X.Y.Z.P'
     - P is some pre-release number.
-9. Send a message to the HDF forum indicating that a pre-release source package is available for testing at <e.g. https://github.com/HDFGroup/hdf5/releases/{hdf5-X.Y.Z-P}> and that feedback from the user community on their test results is being accepted.
+9. Send a message to the HDF forum indicating that a pre-release source package is available for testing at <e.g. //github.com/HDFGroup/hdf5/releases/{hdf5-X.Y.Z-P}> and that feedback from the user community on their test results is being accepted.
```

Comment:
Add a comma:

e.g., 

Is https missing?

Reply:
```suggestion
9. Send a message to the HDF forum indicating that a pre-release source package is available for testing at <e.g. <github.com releases URL>/{hdf5-X.Y.Z-P}> and that feedback from the user community on their test results is being accepted.
```
```suggestion
9. Send a message to the HDF forum indicating that a pre-release source package is available for testing at <e.g. //github.com/HDFGroup/hdf5/releases/{hdf5-X.Y.Z-P}> and that feedback from the user community on their test results is being accepted.
```

Reply:
> Add a comma:
> 
> e.g.,
> 
> Is https missing?

I missed the comma before.  Is the suggestion acceptable, or is there another one that you would prefer?